docs/wiki/99 - Developers/Alternative Install/LORIS on Macintosh/README.md

```diff
@@ -172,9 +172,27 @@ sudo apachectl -k restart
 
 Xdebug allows debugging with breakpoints (such as when using an IDE ex. Intellij).
 
-Type into your terminal and hit enter:
+You will need to install autoconf:
+```
+brew install autoconf
+```
+
+Download the latest Xdebug source from https://xdebug.org
+
+After run the commands as follow in the folder of the source code:
```

Comment:
What source code?

Reply:
Thanks @johnsaigle, latest commit is more descriptive.